scripts/GCE_GPU_Startup_Explained

```diff
@@ -0,0 +1,32 @@
+Setup GPU driver:
```

Comment:
could make this an MD and format the text?

Reply:
Great idea.